machineProblem.py

```diff
@@ -93,6 +93,28 @@ def __init__(self, parent, controller):
         button1 = tkinter.Button(self, height=3, width=20, font=LARGE_FONT, text="Back to Game", command=lambda: controller.show_frame(StartPage))
         button1.pack()
         
+class EasyLevel(tkinter.Frame):
+
+    def __init__(self, parent, controller):
+        tkinter.Frame.__init__(self, parent)
+        label=tkinter.Label(self, text="EASY \n LEVEL", font=font.Font(size=100))
+        label.pack()
+        
+    def show_frame(self, cont):
+        frame = self.frames[cont]
+        frame.tkraise()
+
+        if cont == FirstQuestion:
+            self.after(500, self.show_frame(FirstQuestion))
```

Comment:
source?

Reply:
https://stackoverflow.com/questions/46048610/how-to-change-the-frames-in-tkinter-using-time-and-without-button-command